src/core/test/system/users/borrower.sol

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2019 Centrifuge
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.5.12;
+
+import "ds-test/test.sol";
+import { Title } from "tinlake-title/title.sol";
+import "../interfaces.sol";
+
+contract Borrower is DSTest {
```

Comment:
```suggestion
contract Borrower {
```

If we don't use some log calls it is not required to add it

Reply:
will remove after finishing developing all the tests.